### DISCUSSION TITLE
Please add a "default volume" option so that I don't have to turn every single program down to 25 when launching it for the first time

### DISCUSSION THREAD

**Discussion by kschutter** [File-New-Project/EarTrumpet#960] [Ideas]
^

**Comment by riverar** [comment]
Experimental builds (https://eartrumpet.app) have a feature to perform actions when apps start, stop, etc. Give that a shot?

**Reply by Slegnor** [reply]
Please see bug report [#957](https://github.com/File-New-Project/EarTrumpet/issues/957#issue-1059365732) as actions don't seem to be working
